plugins/katello/nightly/installation/index.md

```diff
@@ -93,9 +93,10 @@ Installation may be done manually or via our recommended approach of using [fork
 {% highlight bash %}
 yum install -y yum-utils
 yum-config-manager --disable "*"
```

Comment:
do we need `yum-config-manager` at all then? seems we could remove that too

Reply:
There is that `yum-config-manager --disable "*"` which is IMO useful/good practice